packages/calcite-components/src/components/fab/fab.e2e.ts

```diff
@@ -148,4 +149,40 @@ describe("calcite-fab", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      themed(html`<calcite-fab></calcite-fab>`, {
+        "--calcite-fab-background-color": {
+          targetProp: "--calcite-button-background-color",
+          shadowSelector: `.${CSS.button} >>> button`,
```

Comment:
These shadowSelectors should not pierce into another shadow DOM.

I think `shadowSelector: `.${CSS.button}`,` should suffice here.

Reply:
Git it! Misunderstood the first time around.